config/webpack/extensions.js

```diff
@@ -0,0 +1,36 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+
```

Comment:
Let's add this linting rule. It will help with the complaints we got from the linter earlier today.

`/* eslint-disable import/no-nodejs-modules */`